src/java/org/apache/cassandra/repair/StreamingRepairTask.java

```diff
@@ -77,10 +80,10 @@ public StreamingRepairTask(RepairJobDesc desc, InetAddressAndPort initiator, Ine
     public void run()
     {
         logger.info("[streaming task #{}] Performing {}streaming repair of {} ranges with {}", desc.sessionId, asymmetric ? "asymmetric " : "", ranges.size(), dst);
-        long start = approxTime.now();
+        long start = approxTime.now(); // TODO (now): mock
```

Comment:
There are a few of these TODOs

Reply:
I leave them to:

1) document work needed
2) make it easy to search for what's remaining

Reply:
removed TODO